book_source/Makefile

```diff
@@ -0,0 +1,5 @@
+all: 
+	Rscript -e 'bookdown::render_book("index.Rmd","bookdown::gitbook", output_dir = "../book")'
```

Comment:
This will need to be changed to reflect what we decide will be our actual build system. Some suggestions on publishing using GitHub [here](https://bookdown.org/yihui/bookdown/github.html).